HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -249,6 +243,21 @@ def replaceItemsInSequence(process, itemsToReplace, replacingSequence):
     itemsList = [process.hltParticleFlowRecHitHBHE, process.hltParticleFlowClusterHBHE,process.hltParticleFlowClusterHCAL]
     process = replaceItemsInSequence(process, itemsList, process.HLTPFHcalClustering)
 
+    process.PFClusterHBHECPUSerial = cms.Sequence(process.hltHBHERecHitToSoA+process.hltPFRecHitSoAProducerHCALCPUSerial+process.hltPFClusterSoAProducerCPUSerial)
```

Comment:
HLT sequences start with uppercase ```HLT``` so rename to ```HLTPFClusterHBHECPUSerial``` and propagate.

Reply:
Right ...

Reply:
Where is this sequence used anyway?

Reply:
The Sequence is then added to `DQM_HcalReconstruction_v6`